api/osrd_infra/tests/test_stdcm.py

```diff
@@ -78,23 +79,26 @@ def setUp(self):
             obj_id="track",
         )
         self.rolling_stock = RollingStock.import_railjson(json.load(open("static/example_rolling_stock.json")))
-        self.schedule = TrainScheduleModel.objects.create(
-            timetable=self.timetable,
+        self.schedule_output = SimulationOutput.objects.create(
+            mrsp=[],
             base_simulation={},
+            eco_simulation={},
+        )
+        self.schedule_input = TrainSchedule.objects.create(
+            timetable=self.timetable,
             departure_time=0,
             initial_speed=0,
-            mrsp=[],
             rolling_stock=self.rolling_stock,
-            eco_simulation={},
+            simulation_output=self.schedule_output,
         )
```

Comment:
`schedule_input` contains the `schedule_output`? Either the naming is off, or something is semantically wrong here.